person.rb

```diff
@@ -1,21 +1,36 @@
-class Person
+require './nameable'
+require './capitalize_decorator'
+require './timmwe_decorator'
```

Comment:
```suggestion
require './trimmer_decorator '
```


- [ ] Kindly make sure you import the trimmer_decorator correctly, this currently throws an error in your terminal